src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -13,9 +13,18 @@
         :next="props.nextUrl !== null"
         :page-offset="props.pageOffset"
         @table-action="($event: any) => selectDataPlaneOverview($event.name)"
-        @load-data="loadData(props.pageOffset)"
+        @load-data="loadData"
       >
         <template #additionalControls>
+          <KFilterBar
+            id="data-plane-proxy-filter"
```

Comment:
Yes, that’s technically possible, but it’s a minor improvement for which I don’t see a pressing need at the moment.

Technically, this would be a utility that creates a string value that’s unique across the application (e.g. `data-plane-proxy-filter-${ID}` or just `${ID}`).